packages/contact-manager-rest-api/src/api-functions.ts

```diff
@@ -0,0 +1,127 @@
+import { checkAuth, sendErrorResponse, ISingleEndpointOpts } from '@sphereon/ssi-express-support'
+import { Request, Response, Router } from 'express'
+import { IRequiredContext } from './types'
+import Debug from 'debug'
+import { AddContactArgs } from '@sphereon/ssi-sdk.contact-manager'
+
+const debug = Debug('sphereon:ssi-sdk:contact-manager-rest-api')
+
+export function partyReadEndpoints(router: Router, context: IRequiredContext, opts?: ISingleEndpointOpts) {
```

Comment:
Please keep the exact structure used elsewhere. There is a reason why there is a function per endpoint! Namely to allow others that do not want to use the class enabling all functions/endpoints. This allows any party to just integrate one or more endpoints in their solution if they desire. With the approach below they cannot decide anymore which endpoints to include, because you will get multiple. 

We will combine certain endpoints in the features that the class is enabling/disabling

Reply:
changed to method per endpoint